app/forms/provider_interface/provider_user_permissions_form.rb

```diff
@@ -36,6 +37,7 @@ def self.from(permissions_model)
     def update_from_params(hash)
       view_applications_only = ActiveModel::Type::Boolean.new.cast(hash[:view_applications_only])
 
+      self.view_applications_only = view_applications_only
```

Comment:
Is there any benefit of doing this here rather than doing the assignment directly on line 38?